app-admin/syslog-ng/syslog-ng-4.3.1.ebuild

```diff
@@ -0,0 +1,186 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
+inherit autotools python-single-r1 systemd
+
+MY_PV_MM=$(ver_cut 1-2)
```

Comment:
I see this variable is used only inside a function in one place. It might be a good idea to just inline the call in there (less subshells in global scope is a good idea).